internal/clients/peering.go

```diff
@@ -78,7 +79,10 @@ func waitForPeeringToBe(state string) WaitFor {
 
 // WaitForPeeringToBePendingAcceptance will poll the GET peering endpoint until
 // the state is PENDING_ACCEPTANCE, ctx is canceled, or an error occurs.
-var WaitForPeeringToBePendingAcceptance = waitForPeeringToBe(PeeringStatePendingAcceptance)
+var WaitForPeeringToBePendingAcceptance = waitForPeeringToBe(PeeringStatePendingAcceptance, []string{PeeringStateCreating})
```

Comment:
Nit: An unfortunate side-effect of the second parameter is we lose some readability (as in, "what are those extra states being passed in?"). I believe the AWS provider just creates a devoted function for each case, but if we want to stick with this parameterized function, we may want to consider passing a config object ie. `{Target: PeeringStatePendingAcceptance, Pending: []string{PeeringStateCreating}}`

Reply:
Ooh I like the config idea